inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -130,11 +132,61 @@ void visit_exec_graph_node(pugi::xml_node& data, std::string& node_type_name,
     }
 }
 
+NodeVector get_ordered_ops(const ngraph::Function& f) {
+    NodeVector op_order;
+    std::stack<std::shared_ptr<Node>> stack;
+    std::unordered_set<Node*> visited;
+
+    auto has_unvisited_inputs = [](const std::unordered_set<Node*>& visited,
```

Comment:
This whole lambda can be implemented with std::any_of the following way:
```
const bool has_unvisited_inputs  = std::any_of(inputs.crbegin(), inputs.crend(), [&visited](const Input<Node>& input) {
    const auto source_output = input->get_source_output().get_node_shared_ptr();
    return visited.count(source_output.get()) == 0;
});
```
I'm not expecting the code to be changed, just a suggestion to use a standard algorithm.

Reply:
Applied in [this](https://github.com/openvinotoolkit/openvino/pull/3156/commits/1b80d2856153c7ccad62f8cd9146934a34529b23)commit.